### ISSUE TITLE
Wrong position of the fixed bottom overlay

### ISSUE THREAD

**Issue by swistach** [handsontable/handsontable#6338]
### Description
If you try to initialize Handsontable without defined width and height, but with a license key then the fixed bottom overlay and its corner are positioning incorrectly.

### Steps to reproduce
1. Initialize Handsontable without width and height but with following settings:
```
licenseKey: 'your-license-key',
data: Handsontable.helper.createSpreadsheetData(2, 1),
fixedRowsBottom: 1
```

### Demo
https://jsfiddle.net/L64k2mvd/

### Your environment
* Handsontable version: 1.14.0+ (I suppose it had been occurring even earlier)
* Browser Name and version: n/a
* Operating System: n/a

**Comment by aninde** [comment]
I found more cases of that issue. fixedRowsBottom doesn't work without declared height.
The behavior varies depending on the type and size of the data.

Check out demo v. latest 7 (`7.2.2`) https://jsfiddle.net/aninde/ecz6fpna/

**Steps to reproduce:** 
Case 1. 
Open demo
Scroll.
![case 1 fixxedbottomrow](https://user-images.githubusercontent.com/30186818/67568926-d45d8b00-f72d-11e9-838c-366f50db6b27.gif)

Case 2.
Comment case 1 and uncomment case 2 in demo.
Scroll.
![case2 fixedbottomRow](https://user-images.githubusercontent.com/30186818/67569051-0ff85500-f72e-11e9-886f-b96f50927544.gif)

**Comment by pnowak** [comment]
Good news! This issue got solved v8.0.0. Here's an updated demo - https://jsfiddle.net/8eua9hzw/
